src/App.tsx

```diff
@@ -1,9 +1,29 @@
-import { Outlet } from 'react-router-dom';
+import SideBar from '@components/SideBar';
+import { useEffect } from 'react';
+import { Outlet, useLocation } from 'react-router-dom';
 
 function App() {
+  const { pathname } = useLocation();
+
+  const bgColor = () => {
+    if (pathname === '/signin' || pathname === '/signup') {
+      return 'bg-white';
+    }
+    return 'bg-slate-200';
+  };
+
+  useEffect(() => {
+    bgColor();
+  }, [pathname]);
```

Comment:
[Important Question] 여기 useEffect 안써도 리렌더 되지 않나요?! 된다면 bgColor를 함수말고 변수로 써도 될 것 같습니다!

```suggestion
  const bgColor = (pathname === '/signin' || pathname === '/signup') ? 'bg-white' : 'bg-slate-200';
  };
```

Reply:
지금 상황에서는 변수로 하는게 더 깔끔할 것 같아요. 반영하도록 하겠습니다!